components/input/input.html

```diff
@@ -0,0 +1,156 @@
+                    </li>
+                    <li class="sidenav-list">
+                        <a href="../../components/modal/modal.html" class="sidenav-item">Modal</a>
+                    </li>
+                    <li class="sidenav-list">
+                        <a href="../../components/navigation/navigation.html" class="sidenav-item">Navigation</a>
+                    </li>
+                    <li class="sidenav-list">
+                        <a href="../../components/rating/rating.html" class="sidenav-item">Rating</a>
+                    </li>
+                </ul>
+            </aside>
+
+            <!-- HERO SECTION -->
+            <div class="content-container">
+                <div class="content">
+                    <h3 class="heading3">Inputs</h3>
+                    <p class="description">
+                        Inputs are used  mostly by forms. 
+                    </p>
+                </div>
+                  
+                <div class="display-component">
+                    <div class="input-container">
+                        <label>Username</label>
+                        <input type="text" class="input-primary">
+                        <label>Email</label>
+                        <input type="text" class="input-email">
+                    </div>
+                </div>
+
+                <div class="code-snippet">
+                    <pre>
+                        <code class="language-html highlight-code">
+                            &lt;label&gtUsername&lt;/label&gt
+                            &lt;input type="text" class="input-primary"&gt
+                
+                            &lt;label&gtEmail&lt;/label&gt
+                            &lt;input type="text" class="input-email"&gt
+                        </code>
+                    </pre>
+                </div>            
+                
+            </div>
+        </div>
+
+        <!-- FOOTER -->
+        <footer class="footer-container flex-column-center">
+            <p class="footer-header">
+            </p>
+            <ul>
+                <li class="footer-item">
+                    <a href="https://github.com/inductionotg" target="_blank">
+                        <i class="fa fa-github footer-icon fa-lg" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li class="footer-item">
+                    <a href="https://twitter.com/riteshs57563347" target="_blank">
+                        <i class="fa fa-twitter footer-icon fa-lg" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li class="footer-item">
+                    <a href="https://www.linkedin.com/in/ritesh-sinha-25822a146/" target="_blank">
+                        <i class="fa fa-linkedin footer-icon fa-lg" aria-hidden="true"></i>
+                    </a>
+                </li>
+            </ul>
+            <p class="copywright">
```

Comment:
I think you meant 'copyright'. The typo might lead to a bit of confusion as to why you have named it this way.